stdlib/priorityqueue.gr

```diff
@@ -0,0 +1,233 @@
+/**
```

Comment:
Can you apply the same changes from the suggestions in immutable to the docs in this file?